target/linux/sunxi/config-4.14

```diff
@@ -361,7 +361,15 @@ CONFIG_MMC_SUNXI=y
 CONFIG_MMU_NOTIFIER=y
 CONFIG_MODULES_TREE_LOOKUP=y
 CONFIG_MODULES_USE_ELF_REL=y
-# CONFIG_MTD is not set
+CONFIG_MTD=y
+CONFIG_MTD_SPLIT_FIRMWARE=y
```

Comment:
Please don't enable this symbol. If you need `firmware` partition to be partitioned, specify a proper `compatible` string in DT, see
[OpenWrt-Devel] Specifying "firmware" partition format on all DTS targets
http://lists.infradead.org/pipermail/openwrt-devel/2018-November/014706.html
for details.

Reply:
Thanks for pointing that out. I did not check split firmware code that well and did not notice this one. I'll remove CONFIG_MTD_SPLIT_FIRMWARE symbol.